SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -36,6 +36,23 @@ void FSpatialGDKEditor::GenerateSchema(FSimpleDelegate SuccessCallback, FSimpleD
 
 	PreProcessSchemaMap();
 
+	// Compile all dirty blueprints
+	TArray<UBlueprint*> ErroredBlueprints;
+	bool PromptForCompilation = false;
+	UEditorEngine::ResolveDirtyBlueprints(PromptForCompilation, ErroredBlueprints);
+
+	if (ErroredBlueprints.Num() > 0)
+	{
+		UE_LOG(LogSpatialGDKEditor, Error, TEXT("There were errors when compiling blueprints. Schema has not been generated."));
+		if (FailureCallback.IsBound())
```

Comment:
Driveby: Could you switch all the IsBound()/Execute() logic in this file to ExecuteIfBound()?